client/flexbox.css

```diff
@@ -0,0 +1,9 @@
+.main__header, .main__div, .title, .nav__menu, .nav__item--sec {
```

Comment:
You don't need to create another css file called `flexbox.css`. Please put your codes on `style.css`